src/tracer.c

```diff
@@ -264,17 +264,6 @@ tracer_main(pid_t pid)
 				  "expected PTRACE_SYSCALL_INFO_ENTRY or PTRACE_SYSCALL_INFO_EXIT\n");
 		    }
 
-		    break;
-		case SIGTRAP:	       // Caused from fork/vfork/clone/.
```

Comment:
why is this change in this PR?

Reply:
@zvr I am kind of confused about it too, wasn't this change already merged in another PR?

Reply:
I merged it and it's now gone. All good